### PR TITLE
Put managing Pinot under in_production

### PR DESCRIPTION
**Pull request by jackjlli**
This PR puts `managing Pinot` under `in_production` page.

### CODE REVIEW

File: docs/in_production.rst [hunk 1]
```diff
@@ -46,17 +46,81 @@ The ordering is as follows:
...
+
+pinot-admin.sh
 --------------
 
-Configuring realtime data ingestion
------------------------------------
+``pinot-admin.sh`` is another way of managing Pinot cluster. This script can be generated by running
+``mvn install package -DskipTests -Pbin-dist`` in the directory in which you checked out Pinot.
+
+For example, to create a pinot segment:
```
Comment (mcvsubbu) [inline docs/in_production.rst]:
```suggestion
Here is an example of invoking the command to create a pinot segment:
```

File: docs/in_production.rst [hunk 2]
```diff
@@ -46,17 +46,81 @@ The ordering is as follows:
...
+  Finished records indexing in IndexCreator!
+  Finished segment seal!
+  Converting segment: /Users/host1/Desktop/test2/baseballStats_data_0 to v3 format
+  v3 segment location for segment: baseballStats_data_0 is /Users/host1/Desktop/test2/baseballStats_data_0/v3
+  Deleting files in v1 segment directory: /Users/host1/Desktop/test2/baseballStats_data_0
+  Driver, record read time : 369
+  Driver, stats collector time : 0
+  Driver, indexing time : 373
+
+To query a table:
```
Comment (mcvsubbu) [inline docs/in_production.rst]:
```suggestion
Here is an example of executing a query on a Pinot table:
```

File: docs/in_production.rst [hunk 3]
```diff
@@ -46,17 +46,81 @@ The ordering is as follows:
...
----------------
+There are two ways to manage Pinot cluster, i.e. using Pinot management console and ``pinot-admin.sh`` script.
 
-Updating tables
----------------
+Pinot Management Console
+------------------------
 
-Uploading data
+There is a REST API which allows management of tables, tenants, segments and schemas. It can be accessed by going to
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply docs/in_production.rst]: Done.

File: docs/in_production.rst [hunk 4]
```diff
@@ -46,17 +46,81 @@ The ordering is as follows:
 Managing Pinot
 ~~~~~~~~~~~~~~
 
-Creating tables
----------------
+There are two ways to manage Pinot cluster, i.e. using Pinot management console and ``pinot-admin.sh`` script.
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply docs/in_production.rst]: Done.

File: docs/in_production.rst [hunk 5]
```diff
@@ -46,17 +46,81 @@ The ordering is as follows:
...
+------------------------
 
-Uploading data
+There is a REST API which allows management of tables, tenants, segments and schemas. It can be accessed by going to
+``http://[controller_host]/help`` which offers a web UI to do these tasks, as well as document the REST API. The below
+is the screenshot of the console.
+
+  .. figure:: img/pinot-console.png
+
+For example, to list all the schemas within Pinot cluster:
```
[the remark this answers is not in the document]
Comment (jackjlli) [reply docs/in_production.rst]: Done.

**Pull request merged**